changelog.d/20221128_151045_nicola.coretti__106_Support_Sqlalchemy_1_4_part_3.rst

```diff
@@ -0,0 +1,36 @@
+.. A new scriv changelog fragment.
+..
+.. Uncomment the header that is right (remove the leading dots).
+..
+🗑️ Removed
+----------
+
+* Removed custom merge statement
```

Comment:
Do we mention, that we plan to introduce a new version compatible with sqla 1.4 later

Reply:
Updated changelog fragment